app/main/views/platform_admin.py

```diff
@@ -105,6 +114,44 @@ def platform_admin_services():
     )
 
 
+@main.route("/platform-admin/admin-actions", endpoint="admin_actions")
+@user_is_platform_admin
+def platform_admin_actions():
+    pending_actions = admin_actions_api_client.get_pending_admin_actions()
+
+    return render_template(
+        "views/platform-admin/admin-actions.html",
+        **pending_actions,
+        permission_labels=dict(permission_options + broadcast_permission_options),
+        key_type_labels=KEY_TYPE_DESCRIPTIONS,
+    )
+
+
+@main.route(
+    "/platform-admin/admin-actions/<uuid:action_id>/review/<new_status>",
+    endpoint="review_admin_action",
+    methods=["POST"],
+)
+@user_is_platform_admin
+def platform_review_admin_action(action_id, new_status):
+    action = admin_actions_api_client.get_admin_action_by_id(action_id)
+
+    if action["status"] != ADMIN_STATUS_PENDING:
+        flash("That action is not pending and cannot be reviewed")
+    elif new_status == "approved" and action["created_by"] == current_user.id:
+        # TODO: Do we turn this off for non-production?
```

Comment:
We should also decide upon this, though my gut says perhaps we should have a setting in the config, and developer environments can have that turned off.

Reply:
I'm drawn to having a config too. For my local needs I ended up just commenting this statement out!

Reply:
I've added a simple config in bd07a98.